unbound.conf

```diff
@@ -24,7 +24,10 @@ server:
   do-daemonize: no
 
   prefetch: yes
+  prefetch-key: yes
+  target-fetch-policy: "-1 -1 -1 -1 -1"
   qname-minimisation: yes
+  harden-referral-path: yes
```

Comment:
This is experimental. I have no experience with this setting and feel a bit uncomfortable setting it.

Reply:
nice to have benefits:
> enforces DNSSEC validation on nameserver NS sets and the nameserver  addresses

according to documentation and as far i understand this, it must be safe. Because just generating some more queries and additional validation for and is only disabled because more queries can produce a bit more load